src/screens/EditProfileScreen.js

```diff
@@ -0,0 +1,129 @@
+
+import React from 'react';
+import { APIRoutes } from '../helpers/routes/routes';
+import BaseRequester from '../helpers/requesters/BaseRequester';
+import {
+  TextInput,
+  Button,
+  ScrollView,
+} from 'react-native';
+
+class EditProfileScreen extends React.Component {
+  constructor(props) {
+    super(props);
+
+  this.state = {
+    updatedUser: this.props.current_veteran,
+  };
+}
+
+
+  _handleUpdate(params) {
+    const successFunc = (responseData) => {
+      this.props.navigation.navigate('ProfileScreen');
+    }
+    updatedUser = this.state;
+    ProfileRequester.updateUser(updatedUser).then(success).catch(failure);
+  }
+
+  render() {
+    // const updated = this.state.updatedUser;
+    <View>
+      <EditProfileForm
+        first_name={this.state.updatedUser.first_name}
+        last_name={this.state.updatedUser.last_name}
+        email={this.state.updatedUser.email}
+        updateSave={this._handleUpdate().bind(this)}  />
```

Comment:
don't call the function when you pass it as a prop to another component. so here it would be `updateSave={this._handleUpdate.bind(this)}`